soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -21,6 +21,7 @@
 start_output = False
 lock = threading.Lock()
 
+VERSION_INFO = "VER 1.0 (ABI V1)"
```

Comment:
What ABI is this?

Reply:
Good question. You are right. This is a not proper name use. It's not binary interface. Will remove it. Just use version to identify is enough.

My original thinking is to tell the user that the client and server should use the same version. Ex.
Assume the V1 support:    --port-req {port}
And the V2 might support it in a different way. (INADDR_ANY, port),

Reply:
The client, server and all other scripts all live in the same repo so I don't see any compatibility problem: people MUST use scripts all from the same commit, end of story. Don't solve problems that you don't have.